sentry_sdk/hub.py

```diff
@@ -110,16 +110,8 @@ def __exit__(self, exc_type, exc_value, tb):
 
 def _check_python_deprecations():
     # type: () -> None
-    version = sys.version_info[:2]
-
-    if version == (3, 4) or version == (3, 5):
-        logger.warning(
-            "sentry-sdk 2.0.0 will drop support for Python %s.",
-            "{}.{}".format(*version),
-        )
-        logger.warning(
-            "Please upgrade to the latest version to continue receiving upgrades and bugfixes."
-        )
+    # Use this to output logger.warnings for deprecated Python versions
+    pass
```

Comment:
I am assuming that we are keeping this method here in case we deprecate any other Python versions in the future?

If so, please update the comment to clarify that we are keeping the method around for future use. With the current comment, the reason why this function still exists is somewhat unclear.

Reply:
Updated, please check.